other_enrichment_methods/nbconverted/3_viz_pathway_ranking.py

```diff
@@ -0,0 +1,223 @@
+
+# coding: utf-8
+
+# # Visualize pathway ranking
+# 
+# This notebook will visualize pathway ranking obtained by the different enrichment analysis methods.
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('load_ext', 'rpy2.ipython')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import pandas as pd
+import numpy as np
+import plotnine as pn
+
+from ponyo import utils
+
+np.random.seed(123)
+
+
+# In[2]:
+
+
+# Read in config variables
+base_dir = os.path.abspath(os.path.join(os.getcwd(), "../"))
+
+config_filename = os.path.abspath(
+    os.path.join(base_dir, "configs", "config_human_general.tsv")
+)
+
+params = utils.read_config(config_filename)
+
+
+# In[3]:
+
+
+# Load params
+dataset_name = params["dataset_name"]
+project_id = params["project_id"]
+
+
+# In[4]:
+
+
+# Create dictionary of enrichment method: statistic
+method_stats_dict = {
+    "GSEA": "padj",
+    "GSVA": "ES",
+    "ROAST": "FDR",
+    "CAMERA": "FDR",
+    "ORA": "p.adjust"
+}
+
+
+# ## Get pathway summary data
+
+# In[5]:
+
+
+# Pathway summary files
```

Comment:
So this requires you to have run the previous notebook for all of the different GSA methods, right? That's fine, but it might be good to make a note of that somewhere (since people might run into errors if they only want to run it for one method, etc)